stories/helpers/formDecorator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Form } from '../../src'
+
+const defaultFormProps = {
+    onSubmit: values => {
+        console.log(
```

Comment:
No need for `join` here:

```js
[
  '++++++++++++++++',
  'Form was submitted with values:',
  JSON.stringify(values, null, 4),
  '----------------',
].map(console.log)
```

Reply:
I tried that, but the effect were very different than what I was looking for.... I was joining strings with a newline so I would end up with a nicely formatted console statement. This is what your suggestion produced:
![Screenshot 2019-08-19 at 14 09 44](https://user-images.githubusercontent.com/353236/63264337-1cbe7f80-c28b-11e9-8091-c962004f0e50.png)
perhaps I can tweak it to prevent multiple lines of console.log, but I will need the newlines for the formatting, so I will keep it the way it was.

Reply:
oh my... I mixed up `lodash/fp`'s map and built-in map.. sorry about that